src/main/java/fi/fmi/avi/model/swx/amd82/SpaceWeatherAdvisoryAmd82.java

```diff
@@ -3,14 +3,13 @@
 import fi.fmi.avi.model.AviationWeatherMessage;
 
 import java.util.List;
-import java.util.Optional;
 
 public interface SpaceWeatherAdvisoryAmd82 extends AviationWeatherMessage {
     IssuingCenter getIssuingCenter();
 
     AdvisoryNumber getAdvisoryNumber();
 
-    Optional<AdvisoryNumber> getReplaceAdvisoryNumber();
+    List<AdvisoryNumber> getReplaceAdvisoryNumber();
```

Comment:
Should we also change the name of this property into the plural form `getReplaceAdvisoryNumbers()`?

Reply:
Yes, changed it.